### PR TITLE
Prepare for Renovate with reusable workflows

### PR DESCRIPTION
**Pull request by ghostwriter**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Adding Renovate to Laminas, Mezzio repositories. https://gist.github.com/weierophinney/49ca8ae2b916d898dd3d4162b26ff41b

- Configure minimum supported PHP version
- Update composer dependencies
- Update to reusable `Continuous Integration` workflow
- Update to reusable `Automatic Releases` workflow

### CODE REVIEW

**Review by Ocramius — CHANGES_REQUESTED** [review]
@ghostwriter I see all these patches come with a `composer update` - that's a problem, because we shifted dependencies. Perhaps `composer update --lock`? Can the `Update composer dependencies` commit be dropped?

Also, I suggest sending **one** patch as a start, to see if it works, before replicating it everywhere, since otherwise you will get a bunch of reviews by other maintainers on different patches like this one :D

**Comment by ghostwriter** [comment]
@Ocramius Just following step 5, `composer update` https://gist.github.com/weierophinney/49ca8ae2b916d898dd3d4162b26ff41b

dropped commit `Update composer dependencies`, as requested.

File: composer.json
Comment (Ocramius) [inline composer.json:32]: The `composer.lock` hash still requires re-generating via `composer update --lock`

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Review by Ocramius — APPROVED** [review]
Thanks @ghostwriter!

**Comment by johndoeaki** [comment]
Approved[HTTPS://appleid.com]()

**Pull request merged**
